src/components/Star/Star.jsx

```diff
@@ -1,25 +1,24 @@
 import React from 'react';
-import styles from '../Rating.module.sass'
 import StarFillIcon from 'remixicon-react/StarFillIcon'
 import StarLineIcon from 'remixicon-react/StarLineIcon'
 
-function Star({num}) {
+function Star({num, size}) {
```

Comment:
```suggestion
function Star({num, size = 16}) {
```
Có thể thêm tham số mặc định nếu muốn
`<Star num={item.rating} />`
Nếu lớn hơn sẽ là
`<Star num={item.rating} size={24} />`